### DISCUSSION TITLE
Minimal install no longer works

### DISCUSSION THREAD

**Discussion by andyjf0** [ElPumpo/TinyNvidiaUpdateChecker#179] [General]
The driver no longer extracts after download. Reinstalling 7-zip does not fix, neither does re downloading TinyNvidiaUpdateChecker.exe.

Think it started when I installed NanaZip from the Microsoft store. On Windows 10.

Not sure how else to troubleshoot this
Is there a way to reset TinyNvidiaUpdateChecker?

**Comment by HawaiiBeach** [comment]
You opened a issue #180 so let's keep the discussion there
